tomviz/OperatorPython.cxx

```diff
@@ -230,7 +230,7 @@ void OperatorPython::setJSONDescription(const QString& str)
   }
 
   m_resultNames.clear();
-  m_childDataSourceNamesAndLabels.clear();
+  m_childDataSourceLabelsAndNames.clear();
```

Comment:
I need a convenient way to map a `DataSource` to its name so that I can use the name as the key to look up the child data value in the returned dictionary. Since the `DataSource` has a label, I made the map go in the opposite direction. Previously, when we set up the child `DataSource` after the script completed, we had both the name and label so we didn't need a map.

I should probably change this to be a map from a `DataSource *` to the name. I'll do that and push another commit.

Reply:
That makes sense.

Reply:
I reverted the change to `m_childDataSourceNamesAndLabels` and create a local map in `OperatorPython::applyTransform()` instead.